devenv/docker/blocks/mariadb/docker-compose.yaml

```
@@ -0,0 +1,7 @@
+  mariadb:
+    image: mariadb:latest
+    restart: always
+    environment:
+      MARIADB_ROOT_PASSWORD: password
```

nice! I think we have to create also a `grafana` user using the `MARIADB_USER` and `MARIADB_PASSWORD` env variables as we do for the [other](https://github.com/grafana/grafana/blob/e1c2eb8c107365188a88e56ad23d912b4663c32e/devenv/docker/blocks/mysql/docker-compose.yaml#L6) SQL devenv blocks

```suggestion
      MARIADB_ROOT_PASSWORD: rootpass
      MARIADB_DATABASE: grafana
      MARIADB_USER: grafana
      MARIADB_PASSWORD: password
```